chapter2/tests/t1/chapter2-t1-two-files.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+output=output
+mkdir -p ${output}
+
+# Create file with base64 string with 700 permission
+# arg1 path
+function createFile {
+  local out=./${output}/$1
+  head -n 10 /dev/urandom | base64 > ${out}
+  chmod 700 ${out}
```

Comment:
Always quote strings containing variables.
```suggestion
  head -n 10 /dev/urandom | base64 > "${out}"
  chmod 700 "${out}"
```